### PR TITLE
fix: sorting select

### LINKED ISSUE #74: Locations sorting not working properly
Actually, the locations sorting it's working. The thing is that the arrow icons - used to indicate which order is current being shown - are not being correctly displayed.

Steps to reproduce:

1. Open the sidebar
2. Sort, for example, ascending
3. Close the sidebar
4. Open the sidebar once again
5. Notice the arrow is facing down, although the sorting it's doing his job as it's ascending

### PR DESCRIPTION
**Pull request by nunom27**
fixes #74

### CODE REVIEW

**Comment by diogogmatos** [comment]
So when I was reviewing this I figured out that I was legit just stupid when I tried to fix this last time because the first time I looked at this now I figured it out immediately, what you did works and is basically the same thing I'm going to suggest you:

So, normally in React when we use a `<select>` (or any other element with an associated value for that matter) we associate its value to a `useState` variable, so the value of the variable and the value of the `<select>` element are linked. Since they weren't, whenever `sortDirection` was changed and we closed and re-opened the Sidebar, the `<select>` value would default to the value of the first `<option>`. The optimal solution is to add `value={sortDirection}` as an attribute of `<select>`. What you did is basically that, but by manually setting that `<option>` as selected, which isn't good practice.

Very well noticed tho ;)

When you can, please make that change :rocket: :raised_hands:

**Review by diogogmatos — CHANGES_REQUESTED** [review]
So when I was reviewing this I figured out that I was legit just stupid when I tried to fix this last time because the first time I looked at this now I figured it out immediately, what you did works and is basically the same thing I'm going to suggest you:

So, normally in React when we use a <select> (or any other element with an associated value for that matter) we associate its value to a useState variable, so the value of the variable and the value of the <select> element are linked. Since they weren't, whenever sortDirection was changed and we closed and re-opened the Sidebar, the <select> value would default to the value of the first <option>. The optimal solution is to add value={sortDirection} as an attribute of <select>. What you did is basically that, but by manually setting that <option> as selected, which isn't good practice.

Very well noticed tho ;)

When you can, please make that change 🚀 🙌

File: components/Sidebar/Sidebar.tsx [hunk 1]
```diff
@@ -126,7 +126,11 @@ export default function Sidebar({ pins, isOpen, setOpen, mapRef }: Props) {
                     className={styles.sort_button}
                   >
```
Comment (diogogmatos) [inline components/Sidebar/Sidebar.tsx:127]:
```suggestion
                    value={sortDirection}
                  >
```

**Review by joaodiaslobo — APPROVED** [review]
Thank you for your contribution!! 🧡🚀

**Pull request merged**